jsonschema-core.md

```diff
@@ -2265,6 +2235,29 @@ determines the canonical nature of the resulting full IRI.[^18]
 and direct you to read the CREF located in the [JSON Pointer fragments and
 embedded schema resources](#embedded) section for further comments.
 
+While the following IRIs do correctly indicate specific schemas, per the reasons
+outlined in {{embedded}}, they are to be avoided:
```

Comment:
I think this language needs to be a little stronger. As a reader, I might assume this statement means it's just poor style and not realize that it there is an interoperability concern.

```suggestion
While the following IRIs do correctly indicate specific schemas, per the reasons
outlined in {{embedded}}, they may not work in all implementations:
```

Reply:
I think stating both is probably good.

```suggestion
While the following IRIs do correctly indicate specific schemas, per the reasons
outlined in {{embedded}}, they are to be avoided as they may not work in all implementations:
```

The "are to be avoided" is quite strong IMO, but I agree with adding _why_.